docs/ref/contrib/gis/geos.txt

```diff
@@ -160,15 +160,18 @@ Geometries support set-like operators::
     The :class:`~GEOSGeometry` equality operator uses
     :meth:`~GEOSGeometry.equals_exact`, not :meth:`~GEOSGeometry.equals`, i.e.
     it requires the compared geometries to have the same coordinates in the
-    same positions::
+    same positions (and additionally, have the same SRIDs)::
 
         >>> from django.contrib.gis.geos import LineString
         >>> ls1 = LineString((0, 0), (1, 1))
         >>> ls2 = LineString((1, 1), (0, 0))
+        >>> ls3 = LineString((1, 1), (0, 0), srid=4326)
         >>> ls1.equals(ls2)
         True
         >>> ls1 == ls2
         False
+        >>> ls3 == ls2  # different SRIDs
+        False
```

Comment:
Add a `.. versionchanged:: 1.11` annotation as described in https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features. What about a mention in the release notes as well?